scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/PaginationView.java

```diff
@@ -77,7 +77,7 @@ public void onClick(ClickEvent event) {
             }
         });
 
-        this.pagePreviousButton = new ToolStripButton("< Previous");
+        this.pagePreviousButton = new ToolStripButton("< Later Jobs");
```

Comment:
I am not sure "later jobs" is a clear label. Then again, I can't think of an optimal label. "Later jobs" seems like jobs that will be handled later.
Maybe "More recent Jobs", but it's too long, or "Newer Jobs"?

Reply:
You are right, I think Newer is better than Later